epics_device/device.py

```diff
@@ -24,8 +26,8 @@ def make(name, address=None, **fields):
 
             # Check for a description, make a report if none given.
             if 'DESC' not in fields:
-                print >>sys.stderr, 'No description for', \
-                    self.separator.join(self.address_prefix + [name])
+                print('No description for', \
+                    self.separator.join(self.address_prefix + [name]), file=sys.stderr)
```

Comment:
Line overrun, please wrap!  Just put the `file=` on the next line under `self.`

Also, the trailing `\` is unnecessary as the expression is inside parentheses.

Reply:
Fixed